Live-Snap/Live-Snap/Extensions/UIImage+.swift

```diff
@@ -23,7 +23,7 @@ extension UIImage {
     }
     
     func resized(size: CGSize) -> UIImage {
-        UIGraphicsBeginImageContextWithOptions(size, false, 1.0)
+        UIGraphicsBeginImageContextWithOptions(size, false, 0.0)
```

Comment:
this shouldn't use 0.0. This won't work if the UIImage's scale isn't set properly. I think...

Instead you should use the UIImage scale and multiply the size property's width/height accordingly to adjust the output image resolution. 

You should check newImage here and see what the resulting scale is being set to if the input image has a different scale than the main screen